modular_citadel/code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -3,6 +3,11 @@
 	build_path = /obj/machinery/vending/kink
 	req_components = list(/obj/item/vending_refill/kink = 3)
```

Comment:
Can you lower this one to 1 as well?

Reply:
Sure thing, I just checked and yeah other vendors do only take one, thanks for pointing that out!